packages/react-priority-overflow/src/useIsOverflowGroupVisible.ts

```diff
@@ -0,0 +1,9 @@
+import { useOverflowContext } from './overflowContext';
+
+/**
+ * @param id - unique identifier for a group of overflow items
+ * @returns visibility state of the group
+ */
+export function useIsOverflowGroupVisible(id: string | number) {
```

Comment:
```suggestion
export function useIsOverflowGroupVisible(id: string | number): boolean {
```

_nit_ Can we make an explicit type declaration?

Reply:
Yes, but this one isn't actually a boolean :)